docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -42,7 +42,14 @@ After downloading, go to your downloads directory, decompress the file and add i
 6- To import gene data type the following commands when in the folder `<your_cbioportal_dir>/core/src/main/scripts`:
 ```
  export PORTAL_HOME=<your_cbioportal_dir>
-./importGenes.pl --genes <ncbi_species.gene_info> --gtf <gencode.v25.annotation.gtf>
+./importGenes.pl --genes <ncbi_species.gene_info> --gtf <gencode.v25.annotation.gtf> --genome-build <GRCh37>
+```
+**IMPORTANT NOTE**: 
+1. The **reference_genome** table needs to be populated before updating the **gene** table. Further details can be found in [this document](import-reference-genome.md). 
+2. Loading genes only to the **reference_genome_gene** table without updating the **gene** table, please use the following command:
+```
+./importGenes.pl --gtf <gencode.v27.annotation.gtf> --genome-build <GRCh38>
+
```

Comment:
Hi @sandertan,
When using a clean database using the seed database, does this include a reference genome? What is the default reference genome?
If the seed database does not include a reference genome, why do I have to add a reference genome to update the genes?
>>> you will need to run the migration script to update you schema. The migration script will populate the reference genome table as well.

As a user, I do not understand the difference between the reference_genome_gene table and gene table. Can you document this somewhere?
>>> how about document in the **updating-gene-and-gene_alias-tables.md** file

Can genes be in the both the reference_genome_gene table and gene table? Which one is used in that case?
>>>yes. Depends on what you are looking for. For **HUGO_GENE_SYMBOL** then reference the **gene** table, for "cytoband", "exonic_length", "start/end" of the gene then reference the **reference_genome_gene** table because those attributes differ between genome builds.

As far as I understand, this PR tries to force the user when updating genes and genomic coordinates, to include a reference genome. This makes a lot of sense. Can you document this reasoning in this markdown?
>>> there is a document **import-reference-genome.md** included in the PR . Can you please check if anything missed?
Thank for help with testing! I will push back those changes based on Pieter comments this morning.